model/vcs/metrics.yaml

```
@@ -0,0 +1,147 @@
+    attributes:
+      - ref: vcs.repository.url.full
+        requirement_level: required
+      - ref: vcs.ref.type
+        requirement_level: required
+  - id: metric.vcs.change.lines_delta
+    type: metric
+    metric_name: vcs.change.lines_delta
+    brief: 'The number of lines added/removed in a change relative to the target branch of the change'
+    instrument: gauge
+    unit: "{line}"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.url.full
+        requirement_level: required
+      - ref: vcs.ref.name
+        requirement_level: required
+      - ref: vcs.ref.type
+        requirement_level: required
+      - ref: vcs.line_change.type
+        requirement_level: required
+  - id: metric.vcs.ref.lines_delta
+    type: metric
+    metric_name: vcs.ref.lines_delta
+    brief: 'The number of lines added/removed in a ref (branch) relative to the previous day'
+    instrument: gauge
+    unit: "{line}"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.url.full
+        requirement_level: required
+      - ref: vcs.ref.name
+        requirement_level: required
+      - ref: vcs.ref.type
+        requirement_level: required
+      - ref: vcs.line_change.type
+        requirement_level: required
+  - id: metric.vcs.change.revisions_delta
+    type: metric
+    metric_name: vcs.change.revisions_delta
+    brief: 'The number of revisions (commits) a change is ahead/behind the target branch of the change'
+    instrument: gauge
+    unit: "{revision}"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.url.full
+        requirement_level: required
+      - ref: vcs.ref.name
+        requirement_level: required
+      - ref: vcs.ref.type
+        requirement_level: required
+      - ref: vcs.revision_delta.type
+        requirement_level: required
+  - id: metric.vcs.ref.revisions_delta
+    type: metric
+    metric_name: vcs.ref.revisions_delta
+    brief: 'The number of revisions (commits) a ref (branch) is ahead/behind the default branch (trunk)'
+    instrument: gauge
+    unit: "{revision}"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.url.full
+        requirement_level: required
+      - ref: vcs.ref.name
+        requirement_level: required
+      - ref: vcs.ref.type
+        requirement_level: required
+      - ref: vcs.revision_delta.type
+        requirement_level: required
+  - id: metric.vcs.ref.time
+    type: metric
+    metric_name: vcs.ref.time
+    brief: 'Time a ref (branch) created from the default branch (trunk) has existed. The `ref.type` attribute will always be `branch`'
+    instrument: gauge
+    unit: "s"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.url.full
+        requirement_level: required
+      - ref: vcs.ref.name
+        requirement_level: required
+      - ref: vcs.ref.type
+        requirement_level: required
+  - id: metric.vcs.repository.contributor.count
```

```suggestion
  - id: metric.vcs.contributor.count
```

I think we can simplify this as well, since an attribute of `repository.url...` will be included, it will always be associated with a repository.

---

That makes sense to me.
I've renamed the metric in b1d8ddc6cff74a46ab74b78c581bbf58949e1e7c.